docs/Options.md

```diff
@@ -207,11 +207,24 @@ Go section is shown only in directories that contain `Godeps`, `glide.yaml`, any
 | Variable | Default | Meaning |
 | :------- | :-----: | ------- |
 | `SPACEFISH_GOLANG_SHOW` | `true` | Show current Go version or not |
-| `SPACEFISH_GOLANG_PREFIX` | `$SPACEFISH_PROMPT_DEFAULT_PREFIX` | Prefix before the Go section |
+| `SPACEFISH_GOLANG_PREFIX` | ` ` | Prefix before the Go section |
```

Comment:
I think you might've unintentionally changed the value here.
The Go prompt prefix is correct to be `$SPACEFISH_PROMPT_DEFAULT_PREFIX`.